src/pages/blog/using-clerk-for-aoc.mdx

```diff
@@ -0,0 +1,185 @@
+---
+author: 'lkn'
+title: 'Using Clerk for Advent of Code'
+description: 'Notebooks <3 Clojure'
+category: 'clojure'
+layout: '../../layouts/BlogPost.astro'
+publishedDate: '23 Dec 2022'
+heroImage: 'clerk.jpg'
+tags:
+  - 'clerk'
+  - 'aoc'
+  - 'clojure'
+---
+
+import { Image } from '@astrojs/image/components'
+import demo from '../../assets/blog/clerk_aoc_demo.png'
+import problem from '../../assets/blog/clerk_aoc_problem.png'
+import css from '../../assets/blog/clerk_aoc_problem_css.png'
+import bad from '../../assets/blog/clerk_aoc_viewer_bad.png'
+import good from '../../assets/blog/clerk_aoc_viewer_good.png'
+
+## Intro
+
+_Just a heads-up, this will contain spoilers for Day 1 of Advent of Code 2022._
+
+_My solutions can be found [here](https://elken.github.io/aoc/)_
+
```

Comment:
```suggestion
_Just a heads-up, this will contain spoilers for Day 1 of Advent of Code 2022. My solutions can be found [here](https://elken.github.io/aoc/)._
```

Reply:
The link gets clipped if I have everything on 1 line